### PR TITLE
Change hovering logic for cell from JS to CSS

### PR DESCRIPTION
**Pull request by aaronraff**
- [x] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)

In response to #1970

### CODE REVIEW

File: packages/desktop/src/notebook/components/cell/cell.js [hunk 1]
```diff
@@ -155,9 +125,7 @@ export class Cell extends React.PureComponent<CellProps, State> {
           this.cellDiv = el;
         }}
       >
-        {cellFocused || hoverCell ? (
-          <Toolbar type={type} cell={cell} id={this.props.id} />
-        ) : null}
+        <Toolbar type={type} cell={cell} id={this.props.id} />
```
Comment (rgbkrk) [inline packages/desktop/src/notebook/components/cell/cell.js:128]: 👍

File: packages/desktop/src/notebook/main.css [hunk 2]
```diff
@@ -268,6 +268,13 @@ body {
   background-color: var(--cell-bg-focus);
 }
 
+/* Show the cell toolbar if hovering on cell,
+   or cell was the last clicked (has .focused class). */
+.cell:hover .cell-toolbar,
+.cell.focused .cell-toolbar {
+  display: block;
```
Comment (rgbkrk) [inline packages/desktop/src/notebook/main.css:275]: There we go, this is great!

**Review by rgbkrk — APPROVED** [review]
Awesome, thank you.

File: packages/desktop/src/notebook/components/cell/cell.js [hunk 3]
```diff
@@ -36,16 +36,11 @@ export type CellProps = {
   models: ImmutableMap<string, any>
 };
 
-type State = {
-  hoverCell: boolean
-};
-
 export class Cell extends React.PureComponent<CellProps, State> {
```
Comment (rgbkrk) [inline packages/desktop/src/notebook/components/cell/cell.js]: There's a current flow error here, you'll want to set this to `React.PureComponent<CellProps, *>`

**Review by rgbkrk — CHANGES_REQUESTED** [review]

**Comment by rgbkrk** [comment]
~Tried this out locally, it _also_ fixes an outstanding bug where the dropdown menu would go transparent and the user wouldn't be able to click some items in the dropdown.~ 🙇 

Edit: Actually, the bug is still there, it's just a bit lessened.

File: packages/desktop/src/notebook/components/cell/cell.js (hunk 3 above)
Comment (aaronraff) [reply packages/desktop/src/notebook/components/cell/cell.js]: Just committed this change, thanks for pointing that out! I'm still pretty new to Flow.

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
This is fantastic, I can't wait for this to go in the next release!

**Comment by aaronraff** [comment]
I'm happy that I could help out, I'm hoping to contribute more in the future!

**Comment by rgbkrk** [comment]
Welcome to the nteract party! 🎉

**Comment by rgbkrk** [comment]
Great, we have much ahead. If you want a next issue to tackle, the dropdown menu could use a fix:

oopsy-under-menu

It's a little better than before, where it was completely unclickable because of the cells underneath hijacking it via that mouse handler. Now at least I think there's just a bit more CSS to get it right.

**Comment by aaronraff** [comment]
I will take a look at that! I'm also thinking that a little CSS will do the trick.

**Comment by aaronraff** [comment]
Took a look at the dropdown menu bug here: #1978

**Pull request merged**